### PR TITLE
docs/api: Add iRMC default port info

### PR DESCRIPTION
**Pull request by zhouhao3**
Signed-off-by: Zhou Hao <zhouhao@fujitsu.com>

### CODE REVIEW

File: docs/api.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ communicate with them.
 | Dell iDRAC      | WSMAN    | iPXE          | `idrac://<host>:<port>`                           |                                                                         |
 |                 | Redfish  | iPXE          | `idrac-redfish://<host>:<port>/<systemID>`        | See above about system ID.                                              |
 |                 | Redfish  | Virtual media | `idrac-virtualmedia://<host>:<port>/<systemID>`   | See above about system ID.                                              |
-| Fujitsu iRMC    | iRMC     | iPXE          | `irmc://<host>:<port>`                            | Port is optional.                                                       |
+| Fujitsu iRMC    | iRMC     | iPXE          | `irmc://<host>:<port>`                            | Port is optional, the default is 443.                                                        |
```
Comment (furkatgofurov7) [inline docs/api.md:40]: Looks like the last `|` shifted a bit more than expected?

**Review by furkatgofurov7 — CHANGES_REQUESTED** [review]

File: docs/api.md (hunk 1 above)
Comment (zhouhao3) [reply docs/api.md]: updated, thanks.

**Comment by zhouhao3** [comment]
@furkatgofurov7 PTAL, thanks.

**Comment by zhouhao3** [comment]
@furkatgofurov7 PTAL, thanks.

**Review by furkatgofurov7 — COMMENTED** [review]
/lgtm

**Pull request merged**